app/controllers/trn_submissions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class TrnSubmissionsController < ApplicationController
+  def show; end
+
+  def create
+    Dttp::ContactService::Create.call(trainee: trainee)
```

Comment:
Is there a chance this could fail? Do we want to handle that here? (or maybe create a separate ticket for it?)

Reply:
Yes. There is a lot of chance. I think this will get moved into a few more layers and get fired off by Sidekiq so I reckon we shouldn't worry about failure for now. Once we flesh it out a bit I'll put a ticket in to sort the error handling out. I don't even know what passes for an error from DTTP. I've only seen 200 and 401 thus far.